src/recorder.js

```diff
@@ -26,6 +28,11 @@ var Recorder = function( config ){
     streamPages: false,
     wavBitDepth: 16,
   }, config );
+
+  if('maxBuffersPerPage' in config && !('maxFramesPerPage' in config)) {
```

Comment:
I'm not particularly interested in maintaining backwards compatibility. We will release a new major version when there are config changes.